### DISCUSSION TITLE
請問如何拿到GIT護身符

### DISCUSSION THREAD

**Discussion by Supermarpig** [ByteLegend/ByteLegend#126] [General]
![image](https://user-images.githubusercontent.com/93064800/160282963-ee6f4167-cab8-49f8-a2b0-76c5c348864e.png)
教程好像消失了~
卡關中><

**Comment by blindpirate** [comment]
Sorry 我们目前还没有git的相关教程。您可以在网上自行搜索相关课程，例如：https://www.youtube.com/results?search_query=%E5%AE%89%E8%A3%85git

**Reply by blindpirate** [reply]
哦……是题目描述消失了，请稍等。

**Reply by blindpirate** [reply]
Google Translation API给出了错误的翻译结果：https://github.com/ByteLegend/game-data/blob/e5a711059066f7ae0e5f8f90b191b2d3ee22ecf9/GitIsland/i18n-auto.json#L330

需要稍后重新发布才能解决，你可以暂时切换至简体中文或者英语。抱歉！

**Comment by desire1992** [comment]
我正在通过游戏学习编程，在爪哇岛的(BJ, 94)坐标处打开
宝箱，输入我的邀请码 c34Nqg5HCIxg74，你和我都可以获得500金币，
然后前往git岛获取
快来 https://bytelegend.com 字节传说加入我吧！

**Reply by desire1992** [reply]
> 我正在通过游戏学习编程，在爪哇岛的(BJ, 94)坐标处打开 宝箱，输入我的邀请码 c34Nqg5HCIxg74，你和我都可以获得500金币， 然后前往git岛获取 快来 https://bytelegend.com 字节传说加入我吧！

有了金币之后，去git岛，执行git --version命令，完成任务并打开第一个宝箱，然后返回java岛
